### PR TITLE
feat(apple): Add info about onCrashedLastRun

### PR DESCRIPTION
**Pull request by philipphofmann**
With sentry-cocoa 6.0.10 we added a callback onCrashedLastRun to the SentryOptions.

### CODE REVIEW

File: src/platforms/apple/common/enriching-events/user-feedback.mdx [hunk 1]
```diff
@@ -31,3 +31,30 @@ userFeedback.email = @"john.doe@example.com";
 userFeedback.name = @"John Doe";
 [SentrySDK captureUserFeedback:userFeedback];
 ```
+
+To capture user feedback of a crash you can use the callback `onCrashedLastRun` of the `SentryOptions`.
```
Comment (marandaneto) [inline src/platforms/apple/common/enriching-events/user-feedback.mdx:35]:
```suggestion
For capturing the User's feedback (only for crashes), you can use the `SentryOptions.onCrashedLastRun` callback.
```

Comment (marandaneto) [reply src/platforms/apple/common/enriching-events/user-feedback.mdx:35]: not sure if it's better, but it reads better on my mind.

Comment (philipphofmann) [reply src/platforms/apple/common/enriching-events/user-feedback.mdx]: I would rather stick to the existing version. Maybe @PeloWriter you can help us here a bit with the wording?

File: src/platforms/apple/common/enriching-events/user-feedback.mdx [hunk 2]
```diff
@@ -31,3 +31,30 @@ userFeedback.email = @"john.doe@example.com";
 userFeedback.name = @"John Doe";
 [SentrySDK captureUserFeedback:userFeedback];
 ```
+
+To capture user feedback for a crash you can use the callback `onCrashedLastRun` of the `SentryOptions`.
```
Comment (PeloWriter) [inline src/platforms/apple/common/enriching-events/user-feedback.mdx:35]:
```suggestion
To capture user feedback regarding a crash, use the callback `onCrashedLastRun` of the `SentryOptions`.
```

Comment (PeloWriter) [reply src/platforms/apple/common/enriching-events/user-feedback.mdx:35]: Let me know if that works better.

**Comment by philipphofmann** [comment]
Thanks @PeloWriter 👏

**Pull request merged**